src/riak_kv_ts_api.erl

```diff
@@ -32,6 +32,7 @@
          get_data/2, get_data/3, get_data/4,
          delete_data/2, delete_data/3, delete_data/4, delete_data/5,
          query/2,
+         row_to_key/3,
```

Comment:
I would rather this function be moved over to `riak_kv_ts_util`. It is not really part of internal API (in the sense `get_data` and the rest are).